README.md

```diff
@@ -131,25 +131,27 @@ preview.show(fileId, accessToken, {
+| viewers                          | {}            | Maps options to a valid viewer type **Use these at your own risk**                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| viewers.Document.disableFontFace | false         | By default fonts are converted to OpenType fonts and loaded via font face rules. If disabled, fonts will be rendered using a built-in font renderer that constructs the glyphs with primitive path commands. **Use this option at your own risk**                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
We should probably also add `viewers.Presentation.disableFontFace`, as that option will also work.